paramiko/hostkeys.py

```diff
@@ -330,7 +330,8 @@ def from_line(cls, line, lineno=None):
         Parses the given line of text to find the names for the host,
         the type of key, and the key data. The line is expected to be in the
         format used by the OpenSSH known_hosts file. Fields are separated by a
-        single space or tab.
+        spaces or tabs between host and key and a single space or tab between
```

Comment:
Typo: Need to remove the ` a` at the end of the preceding line.

Reply:
Fixed